README.asciidoc

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ////
 == TinkerPop3
-
+image::https://codecov.io/gh/Bit-Quill/tinkerpop/branch/master/graph/badge.svg?token=emILhOc6Rm[link="https://codecov.io/gh/Bit-Quill/tinkerpop"]
```

Comment:
Question, will this link have to be edited to point to Apache TinkerPop instead of Bit-Quill, when you PR to TinkerPop?

Reply:
Yes it will need to be changed to apache tinkerpop

Reply:
the token is unaccessible for me, normal I would navigate to [this ](https://app.codecov.io/gh/apache/tinkerpop/settings)page and then the Badges section but I get a 404 if I do.